jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/test/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTasksListGridViewTest.java

```diff
@@ -233,4 +233,15 @@ public void setDefaultFilterTitleAndDescriptionTest() {
         verify(filterPagedTableMock, times(5)).saveTabSettings(anyString(), any(HashMap.class));
     }
 
+    @Test
+    public void initialColumsTest(){
+        view.init(presenter);
+        List<GridColumnPreference> columnPreferences = view.getListGrid().getGridPreferencesStore().getColumnPreferences();
+        assertEquals(columnPreferences.get(0).getName(),COLUMN_NAME);
```

Comment:
expected value should be the first parameter for assertEquals. Please also check code format.